nuget/Microsoft.Windows.CppWinRT.nuspec

```diff
@@ -20,7 +20,6 @@
     <file src="$cppwinrt_exe$" target="bin"/>
     <file src="$cppwinrt_fast_fwd_x86$" target="build\native\lib\win32"/>
     <file src="$cppwinrt_fast_fwd_x64$" target="build\native\lib\x64"/>
-    <file src="$cppwinrt_fast_fwd_arm$" target="build\native\lib\arm"/>
```

Comment:
Are there other source files to delete here?

Reply:
Good question.  I was focused on the nuget and its build breaks but it does seem like there is room for a deeper clean.  Pending.

Reply:
Wow, there was a lot to update to fully remove it.  I just pushed a large iteration that prunes a lot of things (and converts many others from arm32-only to arm64-only).